.config/forest.dic

```diff
@@ -6,6 +6,7 @@ BLS
 CBOR
 CID
 CIDs
+CLI
```

Comment:
I have half a mind to write this out as `command-line` or `command-line interface`.

Reply:
Let's not go this way, otherwise, API may follow. Or even YAML, in which case we're doomed to death by recursion.

Reply:
I'm not sold on the usefulness of `CLI` as a word. Consider the following example:

```rust
/// CLI options
pub struct CliOpts { ... }
```

I don't think this is great documentation. Spelling it out seems slightly better to me:

```rust
/// Command-line options
pub struct CliOpts { ... }
```

The `CLI` word is also used here:

```rust
/// Process CLI subcommand
pub(super) async fn process(command: Subcommand, config: Config) {
```

In this case, I think it would be better to omit it all together:

```rust
/// Process subcommand
pub(super) async fn process(command: Subcommand, config: Config) {
```

I am aware that the documentation is basically useless in both cases but I do think we could try to make the documentation as human readable as possible and avoid acronyms unless they are vital to what is being documented. What are your thoughts on this?

Reply:
It makes sense until you get a really exhaustive, long piece of documentation and then you are forced to use the full form. I'd say we should treat each occurrence separately, even prefer `command line` over `CLI`, but give folks a choice. At the same time, we wrote in this thread more than the total amassed comments including `CLI` in this 3 years-old project, so if you feel it's better then let's do it. :)

Reply:
Okay, we'll keep `CLI` for now and worry about it later when the documentation is re-written.

Reply:
spelling it out sounds better in this case and omitting also just reduces any confusion in cases like `process()`